harmony-api/.env.example

```diff
@@ -1,6 +1,9 @@
 # Port that this server will be accessible on
 PORT=8081
 
+# Endpoint for the frontend
+FRONT_END_BASE_ENDPOINT=http://localhost:8080
+
```

Comment:
I guess that I will need to add variable for this in cd

Reply:
@Dankgen I will add a variable tonight and then merge yours afterwards

Reply:
Yeah, Its needed for the CORS setting

Reply:
> @Dankgen I will add a variable tonight and then merge yours afterwards

Given that you guys find the PR satisfactory, you can merge it after you set it up